api/android/api/jni/Android-nnstreamer.mk

```diff
@@ -0,0 +1,40 @@
+#------------------------------------------------------
+# nnstreamer
+#------------------------------------------------------
+LOCAL_PATH := $(call my-dir)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := nnstreamer
+
+ifndef NNSTREAMER_ROOT
+$(error NNSTREAMER_ROOT is not defined!)
+endif
+
+include $(NNSTREAMER_ROOT)/jni/nnstreamer.mk
+
+LOCAL_SRC_FILES := \
+    $(NNSTREAMER_COMMON_SRCS) \
+    $(NNSTREAMER_PLUGINS_SRCS) \
+    $(NNSTREAMER_CAPI_SRCS) \
+    $(NNSTREAMER_FILTER_TFLITE_SRCS)
+
```

Comment:
If developer want to use decoder, then does he have to add $(NNSSTREAMER_DECODER_BB_SRCS) here and rebuilt it?

Reply:
I will update this later (including all decoders). Single API does not need decoder (appsrc - tensor_filter - appsink is all)

Reply:
I added PR to build static lib with decoders.